go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/knqyf263/go-cpe v0.0.0-20180327054844-659663f6eca2
 	github.com/knqyf263/go-deb-version v0.0.0-20190517075300-09fca494f03d
 	github.com/knqyf263/go-rpm-version v0.0.0-20170716094938-74609b86c936
+	github.com/lib/pq v1.2.0
```

Comment:
pa.Array is required to get the int data type in NomralizedVersion into the database.
we should and will perform a dedicated "convert to pgx" branch and this will quickly go away.